### PR TITLE
Add test to verify update bandwidth to maximum link speed

### PR DESCRIPTION
**Pull request by ylobankov**
The idea of the test is to update bandwidth to maximum link speed without
using alternate links. This test should cover issue #919.

Part of #1085.

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy [hunk 1]
```diff
@@ -162,4 +162,53 @@ class BandwidthSpec extends BaseSpecification {
         cleanup: "Delete created flow"
         flow?.id && northboundService.deleteFlow(flow.id)
     }
+
+    def "Able to update bandwidth to maximum link speed without using alternate links"() {
+        given: "Two active neighboring switches"
+        def switches = topology.getActiveSwitches()
+        def links = northboundService.getAllLinks()
+        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: You can just pick a link and use its src/dst switch.
```
def isl = topology.getIslsOverActiveSwitches().first()
def srcSwitch = isl.srcSwitch
def dstSwitch = isl.dstSwitch
```

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy [hunk 2]
```diff
@@ -162,4 +162,53 @@ class BandwidthSpec extends BaseSpecification {
...
+
+        when: "Create a flow with a valid small bandwidth"
+        def maximumBandwidth = 1000
+
+        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flow.maximumBandwidth = maximumBandwidth
+        northboundService.addFlow(flow)
+
+        then: "Flow is really created and has 'Up' status"
+        Wrappers.wait(5) {
```
Comment (npliashechnikov) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: Maybe it's worth adding assert that checks that flow is actually UP after the wait?

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy [hunk 3]
```diff
@@ -162,4 +162,53 @@ class BandwidthSpec extends BaseSpecification {
...
+        flow.maximumBandwidth = linkSpeed
+        northboundService.updateFlow(flow.id, flow)
+
+        then: "Flow is really updated and has 'Up' status"
+        northboundService.getFlow(flow.id).maximumBandwidth == linkSpeed
+        Wrappers.wait(5) {
+            northboundService.getFlowStatus(flow.id).status == FlowState.UP
+        }
+
+        and: "The same link is used by updated flow"
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: Looking at the code below, I think it's better to say 'the same path'

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy (hunk 2 above)
Comment (npliashechnikov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: Nevermind, looks like "then" clause itself should be enough.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy (hunk 1 above)
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: ok, thanks.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy (hunk 3 above)
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: Fixed.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy (hunk 2 above)
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy]: Yep.

**Comment by sergii-iakovenko** [comment]
@ylobankov Please, check the branches and make sure that you have another PR for func-tests/shorter-longer-flow-path as this one has been merged there.
